src/Extensions/OrderExtension.php

```diff
@@ -311,15 +312,15 @@ public function formatDataForFlow()
 
         // Build up the order
 
-        $xmlOrder = new SimpleXMLElement("<orderHeader/>");
+        $xmlOrder = new SimpleXMLElement('<?xml version="1.0" encoding="UTF-8"?><orderHeader/>');
 
         array_walk($data, function (&$value, &$key) use ($xmlOrder) {
             if (is_numeric($value)) {
                 // Ensure negative values are parsed correctly
                 $value = abs($value);
```

Comment:
Not a part of this PR, but I'm a bit frightened of randomly forcing negative numbers to positive.